### DISCUSSION TITLE
CentOS 9 Stream RPM should be possible now

### DISCUSSION THREAD

**Discussion by burghart** [NCAR/lrose-core#111] [Ideas]
The lack of `netcdf` and `hdf5` RPMs for CentOS 9 Stream had previously stopped my attempts to build lrose-core for that OS. However, the necessary packages have recently been added to the EPEL 9 repository, and I can now build lrose-core.  I have submitted the small changes for CentOS 9 package dependencies as a pull request in the [lrose-bootstrap](https://github.com/ncar/lrose-bootstrap) repo. With that patch in place, the `lrose-bootstrap/scripts/lrose_checkout_and_build.py` builds lrose-core.

Perhaps an RPM package for CentOS 9 Stream can be constructed now?

**Comment by leavesntwigs** [comment]
Hi, Chris.
Thank you for the update.  I see the pull request.
Thanks!
Brenda
